communications/telegram/telegram.go

```diff
@@ -91,17 +91,17 @@ func (t *Telegram) PushEvent(event base.Event) error {
 	msg := fmt.Sprintf("Type: %s Message: %s",
 		event.Type, event.Message)
 
-	var errors error
+	var errs error
```

Comment:
🗻 This is a good opportunity for goros and common.ErrorCollector. Another day though.